### PR TITLE
[REF] Refactor CSS and html masker reports

### PR DESCRIPTION
**Pull request by Remi-Gau**
- Relates to #4368

Changes proposed in this pull request:

- extract CSS of all masker reports into a couple of CSS files so all reports have a similar style

### CODE REVIEW

**Comment by Remi-Gau** [comment]
## Before

https://nilearn.github.io/dev/modules/generated_reports/masker_reports_examples.html

## After

**coming soon**

File: doc/Makefile [hunk 1]
```diff
@@ -174,7 +174,7 @@ doctest:
 	@echo "Testing of doctests in the sources finished, look at the " \
 	      "results in $(BUILDDIR)/doctest/output.txt."
 
-zip: html pdf
```
Comment (Remi-Gau) [inline doc/Makefile:177]: - unrelated but forgotten to do in #4370

**Review by bthirion — APPROVED** [review]
LGTM, thx.

**Review by man-shu — APPROVED** [review]
Looks ok! Thanks!

**Pull request merged**